src/vs/workbench/electron-browser/commands.ts

```diff
@@ -423,3 +424,24 @@ export function registerCommands(): void {
 		});
 	});
 }
+
+namespace GlobalQuickOpenCommand {
```

Comment:
@cristianhosu looks like currently we are not using this explicitly but this test shows how to add a command to the F1 list: https://github.com/Microsoft/vscode/blob/master/src/vs/platform/actions/test/common/menuService.test.ts#L192

Can you add this?